internal/testdb_test_suite.go

```diff
@@ -0,0 +1,95 @@
+package internal
```

Comment:
same question here about the use of the `internal` pkg.

Reply:
It's a test suite which we need to use in common only. Not outside of common.